processors/src/RoryClusterCompareAnaProcessor.cxx

```diff
@@ -0,0 +1,717 @@
+    legend1->AddEntry(times2_,"Times Rec 2");
+    legend1->Draw("same e");
+    c1->SaveAs("alltimes.png");     
+    c1->Clear();
+
+    c1->SetTitle("Times for On Track Clusters");
+    timesOnTrk1_->Draw("e");
+    timesOnTrk2_->Draw("e same");
+    auto legend2 = new TLegend(0.3,0.8,.68,.9);
+    legend2->AddEntry(timesOnTrk1_,"Times On Track Rec 1");
+    legend2->AddEntry(timesOnTrk2_,"Times On Track Rec 2");
+    legend2->Draw("same e");
+    c1->SaveAs("ontimes.png");     
+    c1->Clear();
+    
+    c1->SetTitle("Times for Off Track Clusters");
+    timesOffTrk1_->Draw("e");
+    timesOffTrk2_->Draw("e same");
+    auto legend3 = new TLegend(0.3,0.8,.68,.9);
+    legend3->AddEntry(timesOffTrk1_,"Times Off Track Rec 1");
+    legend3->AddEntry(timesOffTrk2_,"Times Off Track Rec 2");
+    legend3->Draw("same e");
+    c1->SaveAs("offtimes.png");     
+    c1->Clear();       
+    return;
+}
+
+void RoryClusterCompareAnaProcessor::Plot4(){
+    std::cout<<"I AM IN THE FINAL STEP"<<std::endl; 
+    TCanvas *c1 = new TCanvas("c");
+    //gPad->SetLogy(true);
+    c1->cd();
+    
+    positions1_->GetXaxis()->SetTitle("Position");
+    positions1_->GetYaxis()->SetTitle("Hits"); 
+    positionsOnTrk1_->GetXaxis()->SetTitle("Position");
+    positionsOnTrk1_->GetYaxis()->SetTitle("Hits"); 
+
+    positions2_->GetXaxis()->SetTitle("Position");
+    positions2_->GetYaxis()->SetTitle("Hits"); 
+    positionsOnTrk2_->GetXaxis()->SetTitle("Position");
+    positionsOnTrk2_->GetYaxis()->SetTitle("Hits"); 
+
+    positions1_->SetLineColor(kRed);
+    positionsOnTrk1_->SetLineColor(kRed);
+    positions2_->SetLineColor(kBlue);
+    positionsOnTrk2_->SetLineColor(kBlue);
+
+
+    c1->DrawFrame(0.0,3000.0,150.0,7000.0);
+    c1->SetTitle("Position for All Clusters");
+    positions1_->Draw("e");
+    positions2_->Draw("e same");
+    auto legend1 = new TLegend(0.3,0.8,.68,.9);
+    legend1->AddEntry(positions1_,"Position Rec 1");
+    legend1->AddEntry(positions2_,"Position Rec 2");
+    legend1->Draw("same e");
+    c1->SaveAs("positions.png");     
+    c1->Clear();
+
+    c1->SetTitle("Position for On Track Clusters");
+    positionsOnTrk1_->Draw("e");
+    positionsOnTrk2_->Draw("e same");
+    auto legend2 = new TLegend(0.3,0.8,.68,.9);
+    legend2->AddEntry(positionsOnTrk1_,"Position On Track Rec 1");
+    legend2->AddEntry(positionsOnTrk2_,"Position On Track Rec 2");
+    legend2->Draw("same e");
+    c1->SaveAs("onpositions.png");     
+    c1->Clear();
+    
+    c1->SetTitle("Cluster Distances");
+    positionsOnTrk1_->Draw("e");
+    positionsOnTrk2_->Draw("e same");
+    auto legend3 = new TLegend(0.3,0.8,.68,.9);
+    legend3->AddEntry(positionsOnTrk1_,"Cluster Distances Rec 1");
+    legend3->AddEntry(positionsOnTrk2_,"Cluster Distances Rec 2");
+    legend3->Draw("same e");
+    c1->SaveAs("clusDistances.png");     
+    c1->Clear();
+
+    return;
+}
+
+void RoryClusterCompareAnaProcessor::TrackPlot1(){
```

Comment:
what's the difference between trackplot1 and trackplot2? I should be able to tell this from the function name

Reply:
Done